friByte.capture-the-flag.service/friByte.capture-the-flag.service/Services/CtfTaskService.cs

```diff
@@ -196,13 +196,37 @@ private async Task SendSolvedTaskEventToClients(SolvedTask solvedTask)
             return;
         }
 
+        bool isFirstBlood = await IsFirstBlood(solvedTask.TaskId, solvedTask.TeamId);
```

Comment:
Feels a bit weird to run isFirstBlood and then pass the result into `NotifyFirstBlood` without doing any if else first.
- it would make perfect sense if we did not have the `Await Task.WhenAll`.

A bit more cleaner approach would probably be to run `await IsFirstBlood(solvedTask.TaskId, solvedTask.TeamId);` inside `NotifyFirstBlood` on line 221
- Then it would make sense to rename `NotifyFirstBlood` to `MaybeNotifyFirstBlood`